modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaFunctionModelParserTestCase.java

```diff
@@ -33,6 +34,7 @@ public class JavaFunctionModelParserTestCase {
   ClassTypeLoader typeLoader =
       ExtensionsTypeLoaderFactory.getDefault().createTypeLoader(Thread.currentThread().getContextClassLoader());
 
+  @Ignore("W-12398760")
```

Comment:
this looks weird. The issue only talks about `calculation issues`, but the tests are already existing but you're only ignoring them in this PR. Why weren't they failing before?

Reply:
but why ignore them in this PR?. If they're failing at master right now, then they should be ignored in their own PR. If they only fail in this PR, then it's this PR the one breaking them

Reply:
Before this PR, the min mule version calculates was "dead code", meaning, it was only used for its tests. Now that it is being called, the problems for running that code against the test extensions came to float.

I changed it so that for now the calculation just returns empty() and that is why these tests break.

We may have to wait for the bug to get fixed in order to merge this PR.

Reply:
Ok, I understand now. It's fine then, but only as long a retrospective is done with the team to analyze how come this dead yet not working code got merged and how to avoid that in the future. This is a sign of not great coding practices.

@afelisatti ☝🏻

Reply:
We've agreed to not ignore anything. The code needs to be fixed and then Nico's change can go in without the ignores. As for the "dead code", it sounds like some reviews failed.